packages/tree-view-for-tests/src/__tests__/tree.test.ts

```diff
@@ -12,6 +12,9 @@ test('it can serialise a folder', () => {
     ├─ .npmignore #1rstiru
     ├─ CHANGELOG.md
     ├─ README.md #r0jsfd
+    ├─ build
```

Comment:
this shouldn't be committed

Reply:
Yeah - this is annoying I will see if I can ignore this dir so it doesn't keep happening.

Reply:
we technically shouldn't ever have to run a manual build locally on these packages, were you experimenting with something?